posts/month-in-mathlib-may-2024.md

```diff
@@ -0,0 +1,930 @@
+
+<!-- TEASER_END -->
+
+* Algebra
+* [PR #13271](https://github.com/leanprover-community/mathlib4/pull/13271) :: chore(mathlib4/Algebra/BigOperators/Basic): unscope `∑ x ∈ s, f x` notations
+* [PR #12937](https://github.com/leanprover-community/mathlib4/pull/12937) :: feat: Lie algebra root spaces are 1-dimensional
+* [PR #12297](https://github.com/leanprover-community/mathlib4/pull/12297) :: feat(Algebra/Lie): existence of Cartan subalgebras
+* [PR #13307](https://github.com/leanprover-community/mathlib4/pull/13307) :: feat(Algebra/Lie/Weights): The root system of a lie algebra.
+* [PR #12721](https://github.com/leanprover-community/mathlib4/pull/12721) :: feat(Algebra/Category/ModuleCat): the category of presheaves of modules is abelian
+
+* Algebraic Geometry.
+  * [PR #12917](https://github.com/leanprover-community/mathlib4/pull/12917) by Jonas van der Schaaf, Amelia Livingston and later Christian Merten defines closed immersions.
+
+* Analysis.
+  * Sébastien Gouëzel 
+    * [PR #12769](https://github.com/leanprover-community/mathlib4/pull/12769) :: feat: Fourier transform of iterated derivatives
+    * [PR #12144](https://github.com/leanprover-community/mathlib4/pull/12144) :: feat: Fourier transform as a continuous linear equiv on Schwartz space
+  * Chris Birkbeck proved that Eisenstein series converge uniformly in [PR #10377](https://github.com/leanprover-community/mathlib4/pull/10377) and that they are holomorphic in [PR #11013](https://github.com/leanprover-community/mathlib4/pull/11013). This will soon to used to show that they are modular forms.
+
+* Category Theory.
+  * A tremendous amount of derived categories work by Joël Riou is hitting Mathlib at a rapid pace: [PR #12619](https://github.com/leanprover-community/mathlib4/pull/12619) defines t-structures in (pre)triangulated categories, [PR #11738](https://github.com/leanprover-community/mathlib4/pull/11738) and [PR #11786](https://github.com/leanprover-community/mathlib4/pull/11786) show that the localisation of a pretriangulated (resp. triangulated) category by a class of morphisms compatible with the triangulation and having a left (resp. left and right) calculus of fractions is also pretriangulated (resp. triangulated), [PR #11789](https://github.com/leanprover-community/mathlib4/pull/11789) defines the class of morphisms whose cone belongs to a triangulated subcategory, [PR #11805](https://github.com/leanprover-community/mathlib4/pull/11805) defines the long exact sequence of a homological functor from a pretriangulated category to an abelian category, and finally [PR #11806](https://github.com/leanprover-community/mathlib4/pull/11806) constructs the derived category of an abelian category and shows that it is triangulated.
+* [PR #11760](https://github.com/leanprover-community/mathlib4/pull/11760) :: feat: the homological functor on the homotopy category of cochain complexes in an abelian category
```

Comment:
```suggestion
* Category Theory.
  * After an important effort by Joël Riou, derived categories of abelian categories and their triangulated structure are now in Mathlib since [PR #11806](https://github.com/leanprover-community/mathlib4/pull/11806). Using the triangulated structure on the homotopy category of an abelian categories (which was already obtained during the [Liquid Tensor Experiment](https://leanprover-community.github.io/blog/posts/lte-final/)), this essentially follows from the localization theorem for triangulated categories [PR #11786](https://github.com/leanprover-community/mathlib4/pull/11786). Derived categories shall soon have more applications inside mathlib, thanks to the formalization of total derived functors [PR #12788](https://github.com/leanprover-community/mathlib4/pull/12788) and t-structures [PR #12619](https://github.com/leanprover-community/mathlib4/pull/12619).
```